utils/common-config/cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+export default {
+  browser: "chrome",
+  fixturesFolder: false,
+  screenshotOnRunFailure: false,
+  video: false,
+  e2e: {
+	specPattern: "lib/test/unit/frontend/**.test.js",
```

Comment:
This is not a correct value, some tests are in `lib/test/unit` and some in `lib/test/integration`. Either remove this or fix it